drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/BRLConditionColumnSynchronizer.java

```diff
@@ -101,13 +101,15 @@ public boolean handlesUpdate(final MetaData metaData) throws VetoException {
         final Map<String, List<DTCellValue52>> originalColumnsData = new HashMap<>();
         for (BRLConditionVariableColumn variable : originalColumn.getChildColumns()) {
             int iColumnIndex = model.getExpandedColumns().indexOf(variable);
-            final List<DTCellValue52> originalColumnData = new ArrayList<>();
-            final String key = makeUpdateBRLConditionColumnKey(variable);
-            for (List<DTCellValue52> row : model.getData()) {
-                originalColumnData.add(row.get(iColumnIndex));
+            if (iColumnIndex != -1) {
```

Comment:
`iColumnIndex` is `-1` if column is edited second time without saving table in between. This change prevent the `NPE` on the row `originalColumnData.add(row.get(iColumnIndex));`